.pre-commit-config.yaml

```
@@ -17,19 +17,24 @@ repos:
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.0.284
+  rev: v0.6.8
   hooks:
     - id: ruff
       args: [--fix, --exit-non-zero-on-fix]
 
 - repo: https://github.com/psf/black
-  rev: 23.7.0
+  rev: 24.8.0
   hooks:
   - id: black
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)